src/services/formly.providers.ts

```diff
@@ -1,7 +1,9 @@
 import {FormlyPubSub} from "./formly.event.emitter";
 import {FormlyMessages} from "./formly.messages";
+import {SingleFocusDispatcher} from "./formly.single.focus.dispatcher";
 
 export const FormlyProviders = [
   FormlyPubSub,
-  FormlyMessages
+  FormlyMessages,
+  SingleFocusDispatcher
```

Comment:
I kind of agree that SingleFocusDispatcher should be a global level not at individual formly level.

But will global SingleFocusDispatcher cause any problem. if not this looks good.